app/src/main/java/com/bunbeauty/papakarlo/feature/address/screen/user_address_list/UserAddressListFragment.kt

```diff
@@ -51,26 +51,43 @@ class UserAddressListFragment :
         viewModel.update()
         viewBinding.fragmentUserAddressListCvMain.compose {
             val addressListState by viewModel.addressListState.collectAsStateWithLifecycle()
-            UserAddressListScreen(addressListState)
+            UserAddressListScreen(
+                userAddressListState = UserAddressListAndroidState(
+                    userAddressItems = addressListState.userAddressList.map { userAddress ->
+                        userAddressItemMapper.toItem(userAddress)
+                    },
+                    state = addressListState.state,
+                    eventList = addressListState.eventList,
+                ),
+                onCreateAddressClick = viewModel::onCreateAddressClicked,
+                userAddressItemMapper = userAddressItemMapper
```

Comment:
We should't pass mappers to `Composable` functions

Reply:
+